packages/tempus/src/Tempus_StepperNewmarkExplicitAForm_impl.hpp

```diff
@@ -196,7 +196,10 @@ void StepperNewmarkExplicitAForm<Scalar>::setInitialConditions(
     auto f       = initialState->getX()->clone_v();
     this->evaluateExplicitODE(f, x, xDot, initialState->getTime());
     Thyra::Vp_StV(f.ptr(), Scalar(-1.0), *(xDotDot));
-    Scalar reldiff = Thyra::norm(*f)/Thyra::norm(*xDotDot);
+    Scalar reldiff = Thyra::norm(*f);
+    //The following logic is to prevent FPEs  
+    if (Thyra::norm(*xDotDot) > 1.0e-12)
+      reldiff /= Thyra::norm(*xDotDot); 
```

Comment:
Reposting @ccober6's suggestion:

> So I like the compare, but we should probably use a relative compare 
> ```
> if (Thyra::norm(*x) > eps*reldiff) reldiff /= Thyra::norm(*x); 
> ```
> Also I tried to be consistent and use x instead of xDotDot across the Steppers.  That means the output below this needs changing.

I modified these lines because some compilers would have warned on misleading indentation.

Reply:
@ccober6 : I like the relative compare.  I'll go ahead and make the change.  When you say: 

Also I tried to be consistent and use x instead of xDotDot across the Steppers. That means the output below this needs changing.

I think you're referring to this code: https://github.com/trilinos/Trilinos/blob/master/packages/tempus/src/Tempus_StepperNewmarkImplicitAForm_impl.hpp#L296-L307 , is that right?  In particular, 
```
 << "  ||f(x,xDot,xDotDot,t)||/||x|| = " << reldiff << std::endl
```
wouldn't be accurate if `||x|| = 0`, so for that case the print statement needs modification.  Is that what you meant?

Reply:
Yeah.  I will need to go and clean the other steppers.  (Output can be so tedious.  :) )

Reply:
@ccober6 : I see, thanks for clarifying.  If I make the change to the output for the 2 steppers involved in this PR, are you OK merging it in, and having the cleaning up of the output for the other steppers be part of another PR?  I'd like to merge this in sooner rather than later b/c w/o it, there are some failing tests in the Albany nightlies.

Reply:
@ccober6 : I just pushed a fix to do the relative compare as you suggest.  I decided not to change the output at this point - I thought looking at the code and what you said, that if you do a general cleanup of the output that is printed to the screen for all the steppers in a separate PR, it probably makes sense to do this one at that time so that they all look consistent.  I'd like to have this PR pushed to fix the Albany tests failures in the meantime - let me know if you're OK with that.

Reply:
Yes, I was thinking the same thing.